internal/tmpl/tmpl.go

```diff
@@ -82,7 +82,7 @@ func (t *Template) Apply(s string) (string, error) {
 		return "", err
 	}
 
-	sv, err := semver.NewVersion(t.fields[tag].(string))
+	sv, err := semver.NewVersion(t.fields[version].(string))
```

Comment:
this is a good change anyways, can you open a separate PR with just this? Thanks!